src/main/java/ch/njol/skript/SkriptAddon.java

```diff
@@ -87,51 +79,13 @@ public String getName() {
 	 * @return This SkriptAddon
 	 */
 	public SkriptAddon loadClasses(String basePackage, String... subPackages) throws IOException {
```

Comment:
Was there a reason these methods were moved out of here? https://github.com/SkriptLang/Skript/pull/4573 makes a lot of QOL changes here and I'd prefer to be able to keep them.

Reply:
It's moved to the Utils class as TP and I agree it should be so that it can be used in multiple classes. It was moved because we need to be able to register every class inside the JUnit defined packages.

Reply:
Can't we have the JUnit tests refer back to the Skript AddonInstance to load syntax? I'm not sure that those methods should be exposed outside of SkriptAddon

Reply:
We have to be able to collect all the classes.

Reply:
Oh if you have to obtain them that's fine then - I will just update the PR I mentioned to make use of the moved methods